### PR TITLE
Named vips integration test10

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_applications.py [hunk 1]
```diff
@@ -116,80 +116,56 @@ def test_octarine_srv(cluster, timeout=30):
...
+    def _ensure_routable():
+        proxy_uri = 'http://{}:{}/run_cmd'.format(service_points[0].host, service_points[0].port)
+        logging.info('Sending {} data: {}'.format(proxy_uri, cmd))
+        r = requests.post(proxy_uri, data=cmd)
+        logging.info('Requests Response: %s', repr(r.json()))
+        assert(r.json()['status'] == 0)
+    return _ensure_routable
+
+
+def test_if_minuteman_routes_to_vip(registry_cluster):
```
Comment (mellenburg) [inline packages/dcos-integration-test/extra/test_applications.py]: registry_cluster should now be just cluster. registry_cluster is completely gone

**Comment by sargun** [comment]
@spahl please review. @mellenburg too

**Comment by spahl** [comment]
:ship:

File: packages/dcos-integration-test/extra/test_applications.py [hunk 2]
```diff
@@ -116,80 +116,54 @@ def test_octarine_srv(cluster, timeout=30):
     cluster.deploy_marathon_app(app_definition)
 
 
-# By default telemetry-net sends the metrics about once a minute
-# Therefore, we wait up till 2 minutes and a bit before we give up
```
Comment (cmaloney) [inline packages/dcos-integration-test/extra/test_applications.py]: nit: This comment about why the timeout is 125 seems like it's still relevant / useful here

**Comment by cmaloney** [comment]
🚢

**Pull request merged**